common/views/components/LandingBody/LandingBody.js

```diff
@@ -91,12 +91,14 @@ const Body = ({ body, isDropCapped, pageId }: Props) => {
         const hasFeatured =
           Boolean(section.value.hasFeatured) ||
           section.value.items.length === 1;
-        const firstItem = section.value.items[0];
-        const isCardType = firstItem.type === 'card';
+        const firstItem = section.value?.items[0];
```

Comment:
[If `value` isn't nullable](https://github.com/wellcomecollection/wellcomecollection.org/pull/5642#discussion_r508401413), why do we check it? 

Sorry this multi line comment is a bit hard.

Reply:
I think this is down to my misuse of the optional chaining operator. I want to ensure there's at least one item in the `items` array. I think what I need in this case is actually `section.value.items?[0]` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#Syntax

Reply:
We do this in different ways.
```
const first = array.find(Boolean)
const first = array.length > 0 ? array[0] : undefined
```

Saying that, I quite like that syntax - maybe we go with that, which seems to be the equivalent of `.first()` in most other languages.

Bikeshed, as it's not a problem, but if the array started at a non-0 index, this would be an issue... again, it's not.

Reply:
I like the (new) optional-chaining syntax. I'm blaming the fact that it's new that I haven't quite worked out how to use it properly, though! Thanks for picking it up.